projects/packages/doc-parser/src/class-doc-parser.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Package description here
+ *
+ * @package automattic/jetpack-doc-parser
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Converts PHPDoc markup into a template ready for import to a WordPress blog.
+ */
+class Doc_Parser {
+
+	const PACKAGE_VERSION = '1.0.0-alpha';
+
+	/**
+	 * Generate a JSON file containing the PHPDoc markup, and save to filesystem.
+	 *
+	 * @param Array $args this function takes a path as its argument,
+	 * as well as optionally an output file name.
+	 */
+	public function generate( $args ) {
+		list( $directory, $output_file ) = $args;
+
+		if ( empty( $output_file ) ) {
+			$output_file = 'phpdoc.xml';
+		}
+
+		$directory = realpath( $directory );
+		echo PHP_EOL;
+
+		// Get data from the PHPDoc
+		$json = $this->get_phpdoc_data( $directory );
```

Comment:
Is it JSON or XML? If it's JSON, the file name should probably not be "phpdoc.xml". If it's XML, the variable here should probably not be `$json`.

Reply:
Yeah, thanks, resolved!